### PR TITLE
[IMP] turn "Message/datetime" into field

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2793280

### CODE REVIEW

File: addons/mail/static/src/models/message/message.js [hunk 1]
```diff
@@ -331,6 +331,16 @@ registerModel({
         _computeCanStarBeToggled() {
             return !this.messaging.isCurrentUserGuest && !this.isTemporary && !this.isTransient;
         },
+        /**
+         * @private
+         * @returns {string|FieldCommand}
+         */
+        _computeDatetime() {
```
Comment (oomsveta) [inline addons/mail/static/src/models/message/message.js:338]: reorder alphabetically

Comment (alexkuhn) [reply addons/mail/static/src/models/message/message.js:338]: It's alphabetical. `datetime` < `dateDay`.

**Comment by alexkuhn** [comment]
changes  done

**Review by oomsveta — APPROVED** [review]

**Comment by alexkuhn** [comment]
@robodoo r+

**Pull request closed without merging**